cpp/src/ray/runtime/task/task_executor.cc

```diff
@@ -126,7 +126,8 @@ Status TaskExecutor::ExecuteTask(
     const std::vector<ObjectID> &arg_reference_ids,
     const std::vector<ObjectID> &return_ids, const std::string &debugger_breakpoint,
     std::vector<std::shared_ptr<ray::RayObject>> *results,
-    std::shared_ptr<ray::LocalMemoryBuffer> &creation_task_exception_pb_bytes) {
+    std::shared_ptr<ray::LocalMemoryBuffer> &creation_task_exception_pb_bytes,
+    bool *application_level_error_thrown) {
```

Comment:
```suggestion
    bool *application_level_error) {
```

(technically you "raise" in python and not throw, maybe let's avoid the term altogether)

Reply:
Sure. Do we have any coding style about the naming? To me, `application_level_error` variable intuitively holds some error object instead of bool. Maybe `is_application_level_error`?

Reply:
is_application_level_error SGTM!